src/components/pages/community-champions/profile/links-section.js

```diff
@@ -0,0 +1,197 @@
+    margin-bottom: ${size.xsmall};
+    text-transform: uppercase;
+    @media ${screenSize.upToMedium} {
+        font-size: ${fontSize.small};
+        line-height: ${lineHeight.small};
+    }
+`;
+
+const Grid = styled('div')`
+    column-gap: ${COLUMN_GAP};
+    display: grid;
+    grid-template-columns: repeat(2, max-content);
+    margin-bottom: ${size.xlarge};
+    @media ${screenSize.upToSmallDesktop} {
+        grid-template-columns: auto;
+        margin-bottom: ${MOBILE_BOTTOM_MARGIN};
+        row-gap: ${size.large};
+    }
+`;
+
+const BlogsAndPublicationsLinksContainer = styled('div')`
+    display: flex;
+    flex-direction: column;
+    row-gap: ${LINK_ROW_GAP};
+    width: ${LINK_CONTAINER_WIDTH};
+    @media ${screenSize.upToSmallDesktop} {
+        width: 100%;
+    }
+`;
+
+const BlogsAndPublicationsLink = styled(Link)`
+    font-size: ${fontSize.small};
+    line-height: ${lineHeight.small};
+    max-width: max-content;
+`;
+
+const BlogsAndPublications = ({ blogsAndPublications }) => (
+    <div>
+        <Title collapse>Blogs &amp; Publications</Title>
+        <BlogsAndPublicationsLinksContainer>
+            {blogsAndPublications.map(({ id, title, link }) => (
+                <BlogsAndPublicationsLink key={id} href={link} target="_blank">
+                    {title}
+                </BlogsAndPublicationsLink>
+            ))}
+        </BlogsAndPublicationsLinksContainer>
+    </div>
+);
+
+const SocialLinkItemContainer = styled(Link)`
+    align-items: center;
+    column-gap: ${size.xsmall};
+    display: grid;
+    grid-template-columns: ${size.default} auto;
+    text-decoration: none;
+    path {
+        fill: ${({ theme }) => theme.colorMap.devWhite};
+    }
+    &:hover {
+        path {
+            fill: ${({ theme }) => theme.colorMap.darkGreen};
+        }
+    }
+`;
+
+const SocialLinksContainer = styled('div')`
+    column-gap: ${SOCIAL_LINKS_GRID_COLUMN_GAP};
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-columns: repeat(2, max-content);
+    grid-template-rows: repeat(3, auto);
+    row-gap: ${size.xsmall};
+    @media ${screenSize.upToSmallDesktop} {
+        column-gap: ${size.mediumLarge};
+        grid-auto-flow: row;
+        grid-template-columns: repeat(3, max-content);
+        grid-template-rows: repeat(2, auto);
+    }
+    @media ${screenSize.upToSmall} {
+        grid-template-columns: repeat(
+            auto-fill,
+            minmax(${SOCIAL_LINK_MIN_WIDTH}, 1fr)
+        );
+        grid-template-rows: none;
+    }
+`;
+
+const SocialLinkText = styled(P2)`
+    @media ${screenSize.upToMedium} {
+        font-size: ${fontSize.small};
+        line-height: ${lineHeight.small};
+    }
+`;
+
+const SocialLink = ({ icon, link, type }) => (
+    <SocialLinkItemContainer href={link} target="_blank">
+        {icon}
+        <SocialLinkText collapse>{type}</SocialLinkText>
+    </SocialLinkItemContainer>
+);
+
+const Socials = ({ socials }) => {
+    const links = [
```

Comment:
Part of me feels like it is weird we have objects in here we just don't use but honestly this is a pretty clean solution so I am ok with it

Reply:
ahh yeah...it was either this or doing something in the jsx like:
`            {socials.linkedinUrl && <SocialLink icon={<LinkedIn height={16} width={16}/>} type="LinkedIn" link={socials.linkedinUrl}/>}`

and doing that 6 times which felt a bit redundant...what do you think?

Reply:
yeahh we can keep what we have here, but good thinking!